packages/terra-markdown/package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "github-markdown-css": "^2.4.1",
+    "prismjs": "~1.15.0",
```

Comment:
Another dependency, refractor, locks in prismjs like this, I followed suit to avoid duplicate dependencies in terra dev site. I've logged an issue to refractor to update, once they update so can we, but until then this works.